tests/Doctrine/Tests/Common/DataFixtures/Purger/ORMPurgerForeignKeyCheckTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Doctrine\Tests\Common\DataFixtures;
+
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Driver\AbstractMySQLDriver;
+use Doctrine\DBAL\Driver\AbstractSQLiteDriver;
+use ReflectionClass;
+
+/**
+ * @author Robert Freigang <robertfreigang@gmx.de>
+ *
+ * @covers ORMPurger::getTruncateTableSQL
```

Comment:
From https://phpunit.readthedocs.io/en/9.5/annotations.html?highlight=covers#covers:

> Please note that this annotation requires a fully-qualified class name (FQCN). To make this more obvious to the reader, it is recommended to use a leading backslash (even if this not required for the annotation to work correctly).

Also, the table in that documentation says that it is not recommended to specify which method is meant to be covered.

Reply:
Sorry, i forgot running the added test after rebasing. Anyway i fixed it up now.